src/views/Register.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div :class="$style.container">
```

Comment:
Use pug throughout the file, I don't want to put in each line `same here`
```suggestion
  .container
```